rinex/src/record.rs

```diff
@@ -379,32 +379,18 @@ pub fn parse_record(
                 match &header.rinex_type {
                     Type::NavigationData => {
                         let constellation = &header.constellation.unwrap();
-                        if let Ok((e, class, fr)) = navigation::record::parse_epoch(
+                        if let Ok((e, fr)) = navigation::record::parse_epoch(
                             header.version,
                             *constellation,
                             &epoch_content,
                         ) {
-                            if let Some(e) = nav_rec.get_mut(&e) {
+                            if let Some(frames) = nav_rec.get_mut(&e) {
```

Comment:
Maybe try https://doc.rust-lang.org/std/collections/btree_map/enum.Entry.html#method.or_insert_with here and..